run.sh

```diff
@@ -18,7 +19,10 @@ if [ ! -z "$REMOTE_JOB_FILES" ]; then
     done
 fi
 
-# TODO: Parsable JSON output
+# Copy the jobs to the working directory
+cp $JOB_FILES /data
```

Comment:
Would it make sense to create a directory for the benchmark run (using a unique name) and work there?

Reply:
in other words: able to specify the work dir via command line arguments?

Reply:
I don't see the added value of that for such a simple script that usually runs in a container.

Reply:
Added the option anyway :)

Reply:
thank you, very considerate sir!